src/scripts/main.js

```diff
@@ -1 +1,13 @@
 'use strict';
+
+const ul = document.querySelector('ul');
+const target = document.getElementById('largeImg');
+
+ul.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const link = e.target.closest('.list-item__link');
```

Comment:
Ensure that `e.target.closest('.list-item__link')` returns a valid element. If `link` is `null`, calling `getAttribute` on it will cause an error. Consider adding a check to ensure `link` is not `null` before proceeding.